### PR TITLE
[267] add test cases for LanguageValidator

### PR DESCRIPTION
**Pull request by df7734**

### CODE REVIEW

**Review by RockTF — APPROVED** [review]

File: core/src/test/java/greencity/validator/LanguageValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import java.util.Locale;
+
+import static greencity.repository.ModelUtils.getAllLanguages;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+public class LanguageValidatorTest {
```
Comment (nataliiapovzhyk) [inline core/src/test/java/greencity/validator/LanguageValidatorTest.java]: SonarLint point to this public identifier
JUnit5 test classes and methods should have default package visibility

**Review by nataliiapovzhyk — CHANGES_REQUESTED** [review]
fix is needed for SonarLint issue

File: core/src/test/java/greencity/validator/LanguageValidatorTest.java (hunk 1 above)
Comment (df7734) [reply core/src/test/java/greencity/validator/LanguageValidatorTest.java]: Fixed. Thanks for review!

**Review by nataliiapovzhyk — APPROVED** [review]

**Review by ushh789 — APPROVED** [review]
Tests are fine, but please rename the branch according to the convention

**Comment by df7734** [comment]
> Tests are fine, but please rename the branch according to the convention

Fixed. Thanks for review!

**Pull request merged**